src/main/java/com/epam/task/first/StringUtils.java

```diff
@@ -0,0 +1,12 @@
+package com.epam.task.first;
+
+public class StringUtils {
+
+    private final static String ZERO = "0.0";
+    private final static String MINUS = "-";
+
+    public boolean isPositiveNumbers(String str) {
```

Comment:
In the requirements the method has a different name `isPositiveNumber` (without `s`)

Reply:
Corrected, too